### DISCUSSION TITLE
enhancement: preview Image in ComfyUI

### DISCUSSION THREAD

**Discussion by schuster-rainer** [open-webui/open-webui#14296] [Ideas]
**Is your feature request related to a problem? Please describe.**
While genrating an image no preview is displayed

**Describe the solution you'd like**
Comfyui supports sending preview images to the client as callback https://github.com/comfyanonymous/ComfyUI/blob/a38b9b3ac152fb5679dad03813a93c09e0a4d15e/main.py#L151C1-L159C51

**Comment by tjbck** [comment]
PR welcome!
